RELEASE.md

```diff
@@ -1,3 +1,14 @@
+# Release 0.0.20
+
+## New features
+- Custom model pre-loading (https://github.com/marqo-ai/marqo/pull/475). Public models can now be loaded on Marqo startup via URL
```

Comment:
Slight rewording: 
`Public CLIP and OpenCLIP models specified by URL can now be loaded on Marqo startup via ...`